data_structures/trie/trie.py

```diff
@@ -56,20 +85,25 @@ def print_words(node: TrieNode, word: str):  # noqa: E999 This syntax is Python
     :return: None
     """
     if node.is_leaf:
-        print(word, end=' ')
+        print(word, end=" ")
 
     for key, value in node.nodes.items():
         print_words(value, word + key)
 
 
 def test():
-    words = ['banana', 'bananas', 'bandana', 'band', 'apple', 'all', 'beast']
+    words = ["banana", "bananas", "bandana", "band", "apple", "all", "beast"]
     root = TrieNode()
     root.insert_many(words)
```

Comment:
Could we test that __len(the whole tree) == len(words)__?

Could we add a test: __assert all(root.find(word) for word in words)__

Reply:
Can you elaborate on how to go about this?

Reply:
Let's forget the first request because this implementation does not have a `__len__()` method or a `visit()` method that would facilitate the creation of that test.

Reply:
What does the second test imply?

Reply:
__assert all(root.find(word) for word in words)__ will ensure that each word that we have just added can be found.

Reply:
https://docs.python.org/3/library/functions.html#all